### ISSUE TITLE
Assign team

### ISSUE THREAD

**Issue by felixfbecker** [sindresorhus/refined-github#1815]
I would love to have the assignee dropdown show teams, and upon selection, assign all memebers of that team to the issue/PR.

**Comment by bfred-it** [comment]
I don't think we can/should do this, because GitHub doesn't have the concept of "assigned team", so:

- after the team selection we'd have to manually assign all of its members, and that's actually limited to just 10 assignees
- if members are added/removed from the "assigned team," they won't be updated.

I suggest suggesting it to GitHub.

Also, something I read about having multiple assignees: https://help.sifterapp.com/article/43-multiple-assignees

> Sifter is designed with a heavy focus on accountability. In our experience, having multiple assignees dilutes that accountability. By requiring that each issue only have one assignee at a time, there's never any question who is responsible. If that assignee is currently being blocked by a dependency on someone else, they can simply reassign it to that person, and the ball's in that person's court. We've found this keeps things moving forward as each issue becomes a virtual hot potato of sorts.

**Comment by busches** [comment]
At least on GHE I can choose a team to assign it to and everyone gets notified, is there something else here I'm missing?

**Comment by bfred-it** [comment]
@busches that feature doesn't seem to be available on regular GH

**Comment by busches** [comment]
That's pretty weird. It does only work if the team exists in the same Org as the Issue/PR.

**Comment by felixfbecker** [comment]
I am fully aware of the limitations, it would already be a great help to assign all _members_ of a team at the time of assignment, even if they are not synced and even if it's limited to 10 assignees.

**Comment by lukehefson** [comment]
> At least on GHE I can choose a team to assign it to and everyone gets notified, is there something else here I'm missing?

Hey @busches 👋. I think you might be thinking of assigning a team to a _review_ on a pull request – which is a feature across GitHub Enterprise Server, Enterprise Cloud and Team. You currently cannot assign a team using the, more general, <kbd>Assignees</kbd> feature.

It's definitely something that we (GitHub 😄) might be interested in adding though! As ever, we'd love to hear as much as possible about your use case(s) so we can better understand the problem and your needs 💛.

**Comment by busches** [comment]
I am for sure thinking of PRs, but it makes sense that the OP would want it
for issues too.

**Comment by felixfbecker** [comment]
@bfred-it why is this wontfix?

**Comment by bfred-it** [comment]
Because you may know and be ok with the limitations, but others will be confused. I'm leaving this open to possibly gather more information for Luke

**Comment by felixfbecker** [comment]
Why can't we eliminate the confusion by adding a little help text "Assigning a team will assign its members at the time of assignment".

This would save so many seconds and clicks when filing issues every single day.
